rosetta/docs/GPU_performance.md

```diff
@@ -42,17 +42,8 @@ To enable latency hiding optimizations with XLA, turn on the following flag:
 
 To enable asynchronous communication for all collectives, the following is recommended, and is set by default in XLA :
 
-- --xla_gpu_enable_async_collectives=true
 - --xla_gpu_enable_highest_priority_async_stream=true
 
-For more fine-grained control over which collectives should be asynchronous or not, please use: 
```

Comment:
I think we should keep this sentence, but replace the bullets below with `--xla_gpu_disable_async_collectives=allreduce,allgather,reducescatter,collectivebroadcast,alltoall,collectivepermute` or similar.